ci/test/00_setup_env_mac_functional.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
+
+export HOST=x86_64-apple-darwin14
+export BREW_PACKAGES="automake berkeley-db4 libtool boost miniupnpc openssl pkg-config protobuf qt qrencode librsvg pyenv ccache"
```

Comment:
No reason to install `protobuf` if we're not enabling BIP70.

Reply:
Dropped (in other branch, will push later)

Reply:
If we're building/testing only once natively, we should enable everything...

Reply:
I added it back and enabled BIP70 in the build.